src/main/java/com/webperside/courseerpbackend/utils/OTPUtils.java

```diff
@@ -1,12 +1,33 @@
 package com.webperside.courseerpbackend.utils;
 
+import com.webperside.courseerpbackend.exception.OTPValidationException;
+import com.webperside.courseerpbackend.models.enums.otp.OTPValidationResult;
+import com.webperside.courseerpbackend.services.redis.RedisService;
+import org.springframework.stereotype.Component;
+
 import java.util.Random;
 
 public class OTPUtils {
 
+    private static RedisService redisService;
+
+    public static void setRedisService(RedisService service){
```

Comment:
and also add a constructor for injecting RedisService. and take the injected value and set it to the static redis service. we did similar code block on PrivatePublicKeyUtils

Reply:
so you don't need this set method